src/services/prisma-datastore/PrismaAttributeStore.ts

```diff
@@ -0,0 +1,46 @@
+import { PrismaClient } from "@prisma/client";
+import { AttributeStore } from "../AttributeStore";
+
+
+/**
+ * This class wraps a store to also populate 
+ */
+export default class PrismaAttributeStore implements AttributeStore {
+
+    constructor(private prisma: PrismaClient) {}
+
+    // todo - we should probably verify against the project ID?
```

Comment:
How hard would it be to just rip out all the project stuff from prisma since we aren't planning multi-tennancy any time soon? Then at least things would be consistent and there'd be less code to read.

Reply:
not a priority right now